astro/src/components/TipRegistration.astro

```diff
@@ -0,0 +1,49 @@
+---
+---
+
+<form class="col-lg-5 p-4 border-ac tip-form">
+  <p class="display-6">Want a <strong class="text-ac">free and simple</strong> way to learn about accessibility? Sign up for our weekly "Tip of the Week" email to receive quick accessibility tips that can improve your website, social media, events, and more.</p>
+  <p class="display-7">Fill out this short form to sign up. Required fields have an asterisk (<abbr class="display-65 text-ac" title="required">*</abbr>).</p>
+  <div class="row">
+    <div class="col-md-6 mb-3">
+      <label for="firstNameField" class="form-label mb-1 display-65">First Name<abbr class="display-65 text-ac" title="required">*</abbr></label>
+      <input type="text" class="form-control w-100 form-field" id="firstNameField" autocomplete="given-name" required>
+    </div>
+    <div class="col-md-6 mb-3">
+      <label for="lastNameField" class="form-label mb-1 display-65">Last Name</label>
+      <input type="text" class="form-control w-100 form-field" id="lastNameField" autocomplete="family-name">
+    </div>
+  </div>
+    <div class="col mb-3">
+      <label for="emailField" class="form-label mb-1 display-65">Email address <abbr class="display-65 text-ac" title="required">*</abbr></label>
+      <input type="email" class="form-control w-100 form-field" id="emailField" autocomplete="email" required>
+    </div>
+  <div class="row">
+    <div class="col">
+      <button type="submit" class="btn bg-ac text-white display-65 w-100 tip-button">Sign Up</button>
+    </div>
+  </div>
+</form>
+
+<style>
+  /* first declaration that is commented out below is for debugging purposes. */
+  /* * {
+    outline: 1px solid black;
+  } */
+
+  *:focus-visible, .form-field:focus {
```

Comment:
I like what you're doing here, but it's really heavy and is now in a couple of different places. Let's have it align a little more with the `:hover` state and let's put it in a single location.